commands/delete-azure-image.ts

```diff
@@ -0,0 +1,59 @@
+import { Registry } from "azure-arm-containerregistry/lib/models";
```

Comment:
Move this file to a new folder under commands called azureCommands we are standardizing this way

Reply:
done